### PR TITLE
UI - Add 'time' filter

### PR DESCRIPTION
**Pull request by gtrufitt**
Added a new filter for time. The server response contains the data points and x-axis values needed to render the times - and the labels are switched between days and months.

Obviously we don't have that data yet, but shows the ease of adding a 'time' point to the UI and updating the labels and data points according to the period (probably a bit easier than the backend...).

metrics-time-filter

### CODE REVIEW

File: cloudformation/EditorialProductionMetrics.yml [hunk 1]
```diff
@@ -66,6 +66,7 @@ Parameters:
   DBPassword:
     Type: String
     Description: Password for the DB instance
+    NoEcho: true
```
Comment (susiecoleman) [inline cloudformation/EditorialProductionMetrics.yml]: This file has been moved out to editorial-tools-platform repo. But this change is there

File: public/js/components/Charts/Charts.js [hunk 2]
```diff
@@ -1,90 +1,65 @@
...
 import {Grid, Row, Col} from 'react-flexbox-grid';
+import Chart from './Chart';
+
 import ChartTheme from '../ChartTheme/theme';
 
 const customisedTheme = Object.assign({}, Themes.simple, ChartTheme);
 
-const Charts = ({charts}) => {
+const Charts = ({charts, updating, time}) => {
+    const xLabelByTime = time === '7days' || time === '30days'
```
Comment (susiecoleman) [inline public/js/components/Charts/Charts.js]: Is it going to cause issues if we assume all months have 30 days?

File: public/styles/components/_chart.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+$loadingLabelHeight: 30px;
+
+.chart-wrap:after {
+  content: 'Loading\2026'; // Ellipsis
+  display: none;
+  text-align: center;
+  line-height: $loadingLabelHeight;
+  width: $loadingLabelWidth;
+  position: absolute;
+  top: calc(50% - #{$loadingLabelHeight} / 2);
```
Comment (susiecoleman) [inline public/styles/components/_chart.scss]: Does it make sense to try and parameterise this? Would this make it more clear on where these values come from?

File: public/js/components/Charts/Charts.js (hunk 2 above)
Comment (gtrufitt) [inline public/js/components/Charts/Charts.js]: Yes. As per our discussion, I'll swap this out for a date picker and we'll just talk to the api in time periods.

File: public/styles/components/_chart.scss (hunk 3 above)
Comment (gtrufitt) [inline public/styles/components/_chart.scss]: Created a mixin for this.

**Pull request merged**